### ISSUE TITLE
[Tanaris] Jin'zallah the Sandbringer (rare) wrong spawn point

### ISSUE THREAD

**Issue by The-GhostRider** [azerothcore/azerothcore-wotlk#16393]
### Current Behaviour

Jin'zallah the Sandbringer GUID 51829 (https://wowgaming.altervista.org/aowow/?npc=8200) is always spawned at same spot beneath the destroyed tower in Sandsorrow watch.
![jinzallah](https://user-images.githubusercontent.com/80769357/126034344-040df730-16bb-480f-9d76-14fc7015d0ea.jpg)

### Expected Blizzlike Behaviour

Jin'zallah should spawn in one of the tents in Sandsorrow watch.

### Source

https://youtu.be/v9YWJ_iMwxY?t=423

https://www.youtube.com/watch?v=eTWNP6iNSOQ

https://classic.wowhead.com/npc=8200/jinzallah-the-sandbringer#comments

### Steps to reproduce the problem

1. Teleport to GUID 51829 and observe his spawn
2. Check in DB his spawn point locations

### Extra Notes

He is missing movement, but that is blizzlike - he is supposed to be stationary inside the tent he spawns.

2 options for blizzlike spawn points are the tents which also hold solid chest GUID 85717 or 85720, so you can teleport to them and decided on spawn point location.

### AC rev. hash/commit

no mentioned..

### Operating system

Windows 10

**Comment by elthehablo** [comment]
Needs sniff for exact location on wotlk classic. Without it we can get his position close but not exact

**Comment by Kitzunu** [comment]
Correct. Hash/Commit should always be provided or the issue is deemed invalid.
